vector/src/debug/java/im/vector/app/features/debug/features/DebugFeaturesStateFactory.kt

```diff
@@ -28,16 +28,25 @@ class DebugFeaturesStateFactory @Inject constructor(
         return FeaturesState(listOf(
                 createEnumFeature(
                         label = "Onboarding variant",
-                        selection = debugFeatures.onboardingVariant(),
+                        override = debugFeatures.onboardingVariant(),
```

Comment:
I am surprised `override` is not a reserved Kotlin keyword. Maybe a bit confusing to use it anyway.

Reply:
yeah I was surprised as well, will prefix the names to avoid confusion with the language keywords

Reply:
https://github.com/vector-im/element-android/pull/4668/commits/41f931e4663ee3d58cbf5772b5bcee45de10015d